extra/utils/permissions.js

```diff
@@ -0,0 +1,24 @@
+
+const permissions =
+{
+    'getUsers': {
+    all: ['head-trainer'],
+    read : ['trainee', 'trainer'],
+    write : ['trainer'],
+    delete: [],
+}
+
+};
+
+function hasPermission(moduleName, role, permissionType)
+{
+    let data = permissions[moduleName];
+    let x = data[permissionType];
```

Comment:
take the variable name appropriately.
It seems you are not using the Object destructuring to get the values?